src/Symfony/Component/Config/Tests/Resource/DirectoryResourceTest.php

```diff
@@ -96,8 +96,10 @@ public function testIsFreshNewFileWithDifferentPattern()
     public function testIsFreshDeleteFile()
     {
         $resource = new DirectoryResource($this->directory);
+        $time = time();
+        sleep(1);
         unlink($this->directory.'/tmp.xml');
-        $this->assertFalse($resource->isFresh(time()), '->isFresh() returns false if an existing file is removed');
+        $this->assertFalse($resource->isFresh($time), '->isFresh() returns false if an existing file is removed');
```

Comment:
Adding that group causes the test to fail. What does that group do and why would it result in it failing?